README.md

```diff
@@ -39,6 +39,7 @@ ___
   - [User Applications](#User-Applications)
 - [File](#File)
   - [Get a File](#Get-a-File)
+  - [Get a File as base64](#Get-a-File-as-base64)
```

Comment:
wow, look at all this documentation!  How long has this been here?

Reply:
I added it earlier this year. Uses [apidoc](https://www.npmjs.com/package/apidoc) to generate, based on comments in each controller.